newIDE/app/src/ProjectManager/ProjectPropertiesDialog.js

```diff
@@ -420,6 +426,43 @@ class ProjectPropertiesDialog extends React.Component<Props, State> {
               value={adMobAppId}
               onChange={value => this.setState({ adMobAppId: value })}
             />
+            <Text size="title">
+              <Trans>Firebase</Trans>
+            </Text>
+            <SemiControlledTextField
+              floatingLabelText={<Trans>Firebase Configuration string</Trans>}
+              fullWidth
+              rows={8}
+              multiline
+              hintText={`{
```

Comment:
The presence of curly braces inside the text causes an error message to appear in the console (even tho everything works properly). Any Idea on how to fix that?

Reply:
What's this error?

Reply:
The error is: 
```
Message cannot be parsed due to syntax erorrs: {
  apiKey: "AIzaSyBwPHVLhvlHVLh0wqgp4m9qxxxxxxxx",
  authDomain: "xxxxxx-e11a5.firebaseapp.com",
  databaseURL: "https://xxxxx-e11a5.firebaseio.com",
  projectId: "xxxxx-e11xx",
  storageBucket: "xxxxxxx-e11a5.appspot.com",
  messagingSenderId: "254035xxxxxx",
  appId: "1:2345678910:web:2ddd6b830iad9ae3xxxxxxxx",
  measurementId: "G-4REMLXXXXX"
}
```